static/app/components/charts/utils.tsx

```diff
@@ -180,6 +181,23 @@ export function getSeriesApiInterval(datetimeObj: DateTimeObject) {
   return '1h';
 }
 
+export type GranularityStep = [number, string];
```

Comment:
l: Can we make this a named tuple?

```suggestion
export type GranularityStep = [interval: number, label: string];
```

Reply:
Will-do, thanks!